CHANGELOG.md

```diff
@@ -1,5 +1,268 @@
+
+### Security
+
+- [Actions] Bump `autorebase.yml` to `v1.8` ([e944bd7115](https://github.com/facebook/react-native/commit/e944bd711573aab12cccf097daeaae3766d82297) by [@Pranav-yadav](https://github.com/Pranav-yadav))
+- Bump `actions/checkout` to `v3` ([abddf14bf1](https://github.com/facebook/react-native/commit/abddf14bf1797b584bfeff14f7486bfbc114a068) by [@Pranav-yadav](https://github.com/Pranav-yadav))
+
+#### Android specific
+
+
+
+#### iOS specific
+
+
+
+### Unknown
+
+- C++ style enums 7/N: MeasureMode ([3b27c3992a](https://github.com/facebook/react-native/commit/3b27c3992aff523d9809340ce285bc66a3f87024) by [@NickGerleman](https://github.com/NickGerleman))
+- C++ style enums 6/N: PrintOptions ([094046c78e](https://github.com/facebook/react-native/commit/094046c78e249b630f2ee550fd2f6fd50a5af4ea) by [@NickGerleman](https://github.com/NickGerleman))
+- C++ style enums 5/N: LogLevel ([2681b0e2b4](https://github.com/facebook/react-native/commit/2681b0e2b4ee4a59262f6199cc5e1d3c4f72f580) by [@NickGerleman](https://github.com/NickGerleman))
+- C++ style enums 4/N: Errata ([c64c26c3b3](https://github.com/facebook/react-native/commit/c64c26c3b349cbb952fa3408e72423d4d0c0d728) by [@NickGerleman](https://github.com/NickGerleman))
+- C++ style enums 3/N: ExperimentalFeature ([073f5dfe8e](https://github.com/facebook/react-native/commit/073f5dfe8e447b3ce43b46556d7f4a511bafc2a6) by [@NickGerleman](https://github.com/NickGerleman))
+- C++ style enums 2/N: NodeType ([850c93ae6c](https://github.com/facebook/react-native/commit/850c93ae6c5e0f869f6101fc151bc7a69a813632) by [@NickGerleman](https://github.com/NickGerleman))
+- C++ style enums 1/N: Generator ([14bbbdcecc](https://github.com/facebook/react-native/commit/14bbbdceccd9b741a26ca4973d57533e774ed2e8) by [@NickGerleman](https://github.com/NickGerleman))
+- Use fbsource clang-format config ([b29b393d0e](https://github.com/facebook/react-native/commit/b29b393d0e00dddf49023b0cac1bb3c27188789a) by [@NickGerleman](https://github.com/NickGerleman))
+- Remove config drilling ([7f78fa27ae](https://github.com/facebook/react-native/commit/7f78fa27ae37c7cce3c2c16b211d0d708c700fd1) by [@NickGerleman](https://github.com/NickGerleman))
+- Remove layoutContext Drilling ([b955d8d0a7](https://github.com/facebook/react-native/commit/b955d8d0a70193e53b8bee274320c57a96485277) by [@NickGerleman](https://github.com/NickGerleman))
+- Extract isBaselineLayout() ([2d3d3080bb](https://github.com/facebook/react-native/commit/2d3d3080bba7d36baf85c965ae2b8c9c6d5ac7ea) by [@NickGerleman](https://github.com/NickGerleman))
+- Replace dim and pos arrays ([31f51ddb1e](https://github.com/facebook/react-native/commit/31f51ddb1eb87362a6337c19174747ad8212975b) by [@NickGerleman](https://github.com/NickGerleman))
+- Yoga::bit_cast ([8658bdccbc](https://github.com/facebook/react-native/commit/8658bdccbc20d5ae51cbbd7a2b221853df276271) by [@NickGerleman](https://github.com/NickGerleman))
```

Comment:
@NickGerleman are all these commits internal? Can you tell me which commits should be preserved in the changelog and in which section they should go to?